src/cloud-api-adaptor/go.mod

```diff
@@ -57,7 +57,7 @@ require (
 	github.com/sirupsen/logrus v1.9.0
 	github.com/spf13/cobra v1.7.0
 	github.com/tj/assert v0.0.3
-	golang.org/x/crypto v0.16.0
+	golang.org/x/crypto v0.21.0
```

Comment:
Please remove the rebase commit, and resolve the conflicts locally before attaching to the PR. if there is a need to bump the dependencies, we need to describe the reason at the commit level.

Reply:
This is actually due to rebasing... again the rebasing was done in a separate commit leading the reviewers to confusion.
Will be solved when we squash.